cumulus_library/studies/discovery/discovery_templates/code_system_pairs.sql.jinja

```diff
@@ -36,6 +36,7 @@ UNNEST(col_{{ index }}.{{ table.column_hierarchy[index][0].split('.')[0] }}) as
 {%- elif table.column_hierarchy[index][1].__name__ == 'dict' and index +1 == table.column_hierarchy|length -%},
 UNNEST(col_{{ index }}.{{ table.column_hierarchy[index][0].split('.')[0] }}.coding) as table_{{ index +1 }} (col_{{ index +1 }})
 {%- endif %}
+WHERE REGEXP_LIKE({{ get_source_col(table.column_hierarchy) }}.code, '^((?!1.2.840.1.114350).)*$')
```

Comment:
What about this? (code=>system, add urn:oid:, escape dots, add a last dot into the look-behind, move the line start into the look-behind). Have not tested

```
WHERE REGEXP_LIKE({{ get_source_col(table.column_hierarchy) }}.system, '(?!^urn:oid:1\.2\.840\.1\.114350\.).*$')
```

Reply:
Maybe worth adding a unit test

Reply:
i would, but i was trying to get this out before 11am and worry about the testing after